bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/EntryLogScanner.java

```diff
@@ -27,6 +27,10 @@
  * Scan entries in a entry log file.
  */
 public interface EntryLogScanner {
+    public static final int READ_ALL = Integer.MAX_VALUE;
+    public static final int READ_NOTHING = 0;
+    public static final int READ_LEDGER_ENTRY_ID = 16;
```

Comment:
what about `READ_LEDGER_ENTRY_ID_LENGTH` ?

but as we basically expect only these 3 values we can use an "enum" and ensure that the code is not called with other combinations (like negative numbers or non sense numbers caused by bugs)

Reply:
good idea, i have introduced the enum.